modules/custom/az_paragraphs/src/Plugin/paragraphs/Behavior/AZTextWithMediaParagraphBehavior.php

```diff
@@ -83,7 +85,7 @@ public function buildBehaviorForm(ParagraphInterface $paragraph, array &$form, F
       '#options' => [
         'light' => $this->t('Light'),
         'dark' => $this->t('Dark'),
-        'transparent' => $this->t('Transparent'),
+        'bg-transparent' => $this->t('Transparent'),
```

Comment:
Should we remove this as well? https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_paragraphs/css/az_paragraphs_az_text_media.css#L26-L28

Reply:
Sorry, just noticed this comment! That rule seems to be unnecessary but I'm curious to hear what @danahertzberg thinks.

Reply:
Per discussion during the Friday meeting, we will revert the change here from 'transparent' to 'bg-transparent'. With a future issue, we'll restore this change, remove the transparent rule from the CSS file, and require a database update.